### DISCUSSION TITLE
Can a flet application only work "sometimes"?

### DISCUSSION THREAD

**Discussion by mmartinortiz** [flet-dev/flet#1253] [Q&A]
### Question

I am developing a very small application using `flet`. My systems looks like:

- Ubuntu 22.04
- Python 3.10.6
- Flet 0.4.2
- VS Code 1.77.0

On Saturday I was writing some code. It worked, I committed it and pushed it. You can find it [here](https://github.com/mmartinortiz/flet-exif-editor). Switched off the computer. On Monday, I started working again but the following situation happened:

- When using the integrated terminal of VS Code, the `FilePicker` dialog never showed up
- When using the desktop terminal, the `FilePicker` showed up but the selected folder was never returned

On Tuesday (today) I tried again **without changes in my code** and then:

- When using the integrated terminal of VS Code, the FilePicker dialog never showed up
- When using the desktop terminal, everything works perfectly, as it happened on Saturday

This is a situation that I do not understand. When running the application with `flet -vv` nothings is shown in the terminal. Moreover, I discovered another difference regarding Saturday:

- On VS Code, the `initial_directory` property is ignored
- On the terminal, the `initial_directory` property works as expected

There have been other updates on the system, but those are normal updates from Ubuntu.

What would be the reasons for this situation to happen?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Sorry, you are running on Ubuntu. I've deleted my comment.

**Reply by mmartinortiz** [reply]
Today I got back to work on my project. No code has been changed since last time, that is when I open this discussion. From VS Code, the `FilePicker` dialog does not open. From the normal terminal the `FilePicker` dialog opens, I can select a folder, but the folder path is not returned to the main program.

The flag `-vv` does not show any output.

Can you reproduce this situation? or is it just something that happens to me?
